Habitica/res/values/strings.xml

```diff
@@ -860,5 +861,12 @@
     <string name="discover">Discover</string>
     <string name="damage_paused">Damage paused</string>
     <string name="preference_push_important_announcements">Important Announcements</string>
+    <string name="no_notifications_title">You’re all caught up!</string>
+    <string name="no_notifications_text">The notification fairies give you a raucous round of applause! Well done!</string>
+    <string name="dismiss_all">Dismiss All</string>
+    <string name="new_bailey_update">New Bailey Update!</string>
+    <string name="new_msg_guild"><![CDATA[<b>%1$s</b> has new posts]]></string>
+    <string name="new_msg_party"><![CDATA[Your Party, <b>%1$s</b>, has new posts]]></string>
+    <string name="unallocated_stats_points"><![CDATA[You have <b>%1$s unallocated Stat Points</b>]]></string>

```

Comment:
It's ok to use `Html.fromHtml` with these, or is there some better way to achieve same thing?

Reply:
I haven't really found a better way to do this yet, so jeah, that's ok.